style.css

```diff
@@ -101,4 +101,27 @@ button {
     text-transform: uppercase;
     background: var(--accent-color);
     color: #fff;
+}
+
+
+/*
+**Desktop draft
+*/
+
+@media only screen and (min-width: 961px) {
+    section {
+        margin: 3rem;
+        padding: 3rem;
```

Comment:
Instead of playing with the margins, we can set a max-width and center the section. Something like this:

```css
max-width: 900px;
margin: 0 auto;
```

Reply:
Ok i'll try this :)